libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -316,7 +327,7 @@ export class LuisRecognizer implements LuisRecognizerTelemetryClient{
      * @param telemetryProperties Additional properties to be logged to telemetry with the LuisResult event.
      * @param telemetryMetrics Additional metrics to be logged to telemetry with the LuisResult event.
      */
-    protected onRecognizerResults(recognizerResult: RecognizerResult, turnContext: TurnContext, telemetryProperties?: {[key: string]:string}, telemetryMetrics?: {[key: string]:number}): Promise<void> {
+    protected async onRecognizerResults(recognizerResult: RecognizerResult, turnContext: TurnContext, telemetryProperties?: {[key: string]:string}, telemetryMetrics?: {[key: string]:number}): Promise<void> {
         this.fillTelemetryProperties(recognizerResult, turnContext, telemetryProperties).then(props => {
```

Comment:
think we need an `await` for `this.fillTelemetryProperties()`

Reply:
Looks like you did await on the C# side too 
https://github.com/Microsoft/botbuilder-dotnet/blob/5b3075cbac8a2c15d953282f472ced63dd6f931b/libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizer.cs#L190

Reply:
Great catch!!